dist/osel-search.js

```diff
@@ -41,6 +41,17 @@
         }, {});
         // ---------- variables setup end -----------
 
+          var observableWithPreTransformResult = function(preTransformResult, term) {
+              return rx.Observable.create(function (observer) {
+                  try {
+                      var result = preTransformResult.call(this, term);
+                          observer.onNext(result);
```

Comment:
No reason I just forgot to call it. I will add it in as I think this would mean this observable would never cleanly be closed.